types/address_test.go

```diff
@@ -37,70 +30,29 @@ func testMarshal(t *testing.T, original interface{}, res interface{}, marshal fu
 }
 
 func TestEmptyAddresses(t *testing.T) {
-	require.Equal(t, (types.AccAddress{}).String(), "")
-	require.Equal(t, (types.ValAddress{}).String(), "")
-	require.Equal(t, (types.ConsAddress{}).String(), "")
+	require.Equal(t, (types.Address{}).String(), "")
```

Comment:
Looks like a lot of redundancy in these tests, I'm going to post a new issue so we can get this one merged asap